internal/controller/operatorconfigmap_controller.go

```diff
@@ -700,13 +698,15 @@ func (c *OperatorConfigMapReconciler) reconcileClientOperatorSubscription() erro
 }
 
 func (c *OperatorConfigMapReconciler) reconcileNoobaaOperatorSubscription() error {
-	noobaaSubscription, err := c.getSubscriptionByPackageName("noobaa-operator")
+	noobaaSubscription, err := c.getSubscriptionByPackageName("mcg-operator")
+	if kerrors.IsNotFound(err) {
+		noobaaSubscription, err = c.getSubscriptionByPackageName("noobaa-operator")
+	}
 	if err != nil {
 		return err
 	}
-	desiredSubChannel := utils.GetNoobaaOperatorChannel()
-	if desiredSubChannel != noobaaSubscription.Spec.Channel {
-		noobaaSubscription.Spec.Channel = desiredSubChannel
+	if c.subscriptionChannel != "" && c.subscriptionChannel != noobaaSubscription.Spec.Channel {
+		noobaaSubscription.Spec.Channel = c.subscriptionChannel
```

Comment:
The error mesasge at line 711 need to be updated to point to mcg-operator as well, currently its pointed to noobaa-operator only or is this fine?

Reply:
ah, it's u/s vs d/s, will change the preference to d/s.